bin/validate-json

```diff
@@ -218,7 +218,8 @@ $refResolver = new JsonSchema\RefResolver($retriever);
 $refResolver->resolve($schema, $urlSchema);
 
 if (isset($arOptions['--dump-schema'])) {
-    echo json_encode($schema, JSON_PRETTY_PRINT) . "\n";
+    $options = version_compare(PHP_VERSION, '5.4.0', '>=') ? JSON_PRETTY_PRINT : 0;
```

Comment:
You want to check if sth you want to use is available for you.
Therefore please check that sth, not sth other. When you want to have `JSON_PRETTY_PRINT` you should test for it, not for some PHP_VERSION. Remember that one may want to run it under hhvm, and it has it's own problem with constants (they forgot to create some of them).

Ref: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.7/Symfony/CS/Utils.php#L30